test/script-loader.js

```diff
@@ -759,6 +759,27 @@ describe('Loader', function() {
         }, 50);
     });
 
+    it('should implement localRequire.toUrl inside module implementation function', function(done) {
+        var failure = sinon.spy(function(error) {
+            console.error(error);
+        });
+        var success = sinon.spy(function(module) {
+            this.module = module;
+        });
+
+        Loader.require(['module-require'], success, failure);
+
+        setTimeout(function() {
+            assert.isTrue(failure.notCalled, 'Failure should not be called');
+            assert.isTrue(success.calledOnce, 'Success should be called once');
+            assert.isTrue(
+                success.module.resolvedUrl.endsWith('test/fixture/modules/module1.js'),
```

Comment:
@jbalsas Because in the tests the URL is resolved to the absolute path of the module that usually begins with your home folder and project's directory. We could also compose the path getting those values, but endsWith() should work well too. We can change it, if you find it clearer...